network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -184,7 +185,7 @@ class ProductPage(FoundationMetadataPageMixin, Page):
     )
     review_date = models.DateField(
         help_text='Review date of this product',
-        auto_now_add=True,
+        default=timezone.now
```

Comment:
by using `auto_now_add` this field had become read-only, which meant we couldn't actually copy over values from real products. This should do the same thing, but allows the field to be set to whatever we need.